scripts/package/run-sdist-test.sh

```diff
@@ -5,9 +5,9 @@
 set -e
 
 rm -rf dist
-python3 setup.py sdist
+python3 -m build --sdist
```

Comment:
I saw we have multiple occurrences of `python3`, since we require Python >= 3 anyway, I guess we can just renamed those.
What do you think?

Reply:
Yes, I think this is true. We have this a lot on the mac build also. Not sure there. But my preferred way would be to get rid of all the python3 calls. I changed it to just "python" here in this file now. Let's do this more generally in a separate PR.